<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1125</width>
    <height>856</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Art-Ifex</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/newPrefix/blueprism.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox1l">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>451</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Input Selection</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="image_select_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="accessibleDescription">
      <string/>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/pre/C:/Users/JESWIN/Desktop/PyQt Tutorial/image icon.jpg);</string>
     </property>
     <property name="text">
      <string>Select Image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="video_select_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/pre/C:/Users/JESWIN/Desktop/PyQt Tutorial/videofileicon.png);</string>
     </property>
     <property name="text">
      <string>Select Video</string>
     </property>
    </widget>
    <widget class="QLabel" name="label1">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>191</width>
       <height>81</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>No Image selected</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>140</y>
       <width>191</width>
       <height>81</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>No Video selected</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="Output_location_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/pre/C:/Users/JESWIN/Desktop/PyQt Tutorial/videofileicon.png);</string>
     </property>
     <property name="text">
      <string>Save Output As</string>
     </property>
    </widget>
    <widget class="QLabel" name="label3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>250</y>
       <width>191</width>
       <height>81</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Output File path not selected</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <zorder>video_select_button</zorder>
    <zorder>image_select_button</zorder>
    <zorder>label1</zorder>
    <zorder>label2</zorder>
    <zorder>Output_location_button</zorder>
    <zorder>label3</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox2r">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>140</y>
      <width>421</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/newPrefix/white background.jpg);</string>
    </property>
    <property name="title">
     <string>Weight Selection</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QGroupBox" name="groupBox_3u_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>321</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/pre/C:/Users/JESWIN/Desktop/PyQt Tutorial/image icon.jpg);</string>
      </property>
      <property name="text">
       <string>Select Style weight</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="style_weight_spinner">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/pre/C:/Users/JESWIN/Desktop/PyQt Tutorial/image icon.jpg);</string>
      </property>
      <property name="maximum">
       <double>1000000.000000000000000</double>
      </property>
      <property name="value">
       <double>99999.000000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4d">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>321</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/pre/C:/Users/JESWIN/Desktop/PyQt Tutorial/image icon.jpg);</string>
      </property>
      <property name="text">
       <string>Select Content weight</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="content_weight_spinner">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/pre/C:/Users/JESWIN/Desktop/PyQt Tutorial/image icon.jpg);</string>
      </property>
      <property name="maximum">
       <double>1000000.000000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="submit_button">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>560</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newPrefix/button_submit1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>670</y>
      <width>621</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="output_button">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>560</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newPrefix/button_get_output.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1125</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="dummy.qrc"/>
 </resources>
 <connections/>
</ui>
